omnibus/config/software/opscode-solr4.rb

```diff
@@ -42,10 +42,10 @@
 build do
   env = with_standard_compiler_flags(with_embedded_path)
 
-  embedded_jre_path = "#{install_dir}/embedded/jre/bin"
```

Comment:
micro nitpick; I might leave the variable name alone, since it's still the JRE path whether it's OpenJRE or or not.